src/module/entities/helper.js

```diff
@@ -580,18 +581,26 @@ module.exports = class UserProjectsHelper {
 
 					// Prepare registryDetails based on singleEntity data
 					let registryDetails = {}
-					if (singleEntity.locationId) {
-						registryDetails['locationId'] = singleEntity.locationId
-						if (singleEntity.code) {
-							registryDetails['code'] = singleEntity.code
+					if (singleEntity.externalId) {
```

Comment:
@MallanagoudaB update validator to make sure externalId is passed. then you don't need this check